packages/flutter_tools/lib/src/commands/build_aar.dart

```diff
@@ -111,6 +111,15 @@ class BuildAarCommand extends BuildSubCommand {
     await super.validateCommand();
   }
 
+  @override
+  Future<void> regeneratePlatformSpecificTooling(
+    FlutterProject project, {
+    bool? releaseMode,
+  }) async {
+    // Intentionally left blank. We want to rebuild the tooling before every sub-build.
```

Comment:
uh ... can you explain this to me?

Reply:
Yeah. The base class implementation is automatically called in verifyAndRunCommand, but I don't want that to happen in these sub commands. I admit it could probably be done a better way, I was just trying to avoid lots of overridden booleans

Reply:
oh, so the base class is the real thing that we use before each build. BUt then there is also a verfy step that we need to .. for lack of a better term, trick?

If my understanding is correct, we could also override/remove the verify step too right?

Reply:
That's essentially what this does, at least the part related to meta tooling. I'm not sure it's safe to skip the entire verify step, but I could be talked into it by someone who understands what it does better 😁

Reply:
Yeah this seems simpler, I would expand the documentation to explain it more directly.

Reply:
Done. I ended up also renaming the function and adding the boolean 🤷🏼